src/components/Dashboard/AddressListSm.vue

```diff
@@ -107,9 +107,9 @@
         </v-expansion-panel-content>
       </v-expansion-panel>
 
-      <!-- Records Office (BCOMPs and CORPs) -->
+      <!-- Records Office (BEN/BC/CCC/ULC and CORPs) -->
       <v-expansion-panel id="records-office-panel"
-        v-if="isBComp || isCorp"
+        v-if="isBenBcCccUlc || isCorp"
```

Comment:
This is the fix for bcgov/entity#14322.

![image](https://user-images.githubusercontent.com/10002889/202822531-57396714-9c49-47f5-80cd-0a2ebdff0114.png)